.gitignore

```diff
@@ -1,5 +1,2 @@
 /build
 /node_modules
-
-/bindings/rust
-/Cargo.toml
```

Comment:
Do we need to add `Cargo.lock` file? If not, we can add it to `.gitignore` file to prevent someone committing it.

Reply:
.gitignore